README.md

```diff
@@ -202,16 +202,10 @@ gcloud builds submit --config=cloudbuild.yaml
 Navigate to the
 [agones-deploy-pipeline](https://console.cloud.google.com/deploy/delivery-pipelines/us-central1/agones-deploy-pipeline)
 delivery pipeline to review the rollout status. Cloud Build will create a Cloud Deploy release which automatically
-deploys Agones the first game server cluster. Agones can be deployed to subsequent clusters by clicking on the
+deploys Agones to all of the game server clusters automatically. Agones can be deployed manually to subsequent clusters by clicking on the
```

Comment:
This line should read:

```
deploys Agones to all of the game server clusters sequentially, with a (by default) 300s wait in between using [Cloud Deploy Automations](https://cloud.google.com/deploy/docs/automation).
```

Sorry, I wasn't clear in my previous comment (not sure why I can't make the suggestion)